Backend/Controllers/Members.cs

```diff
@@ -0,0 +1,513 @@
+        /// <returns>Nothing, really.</returns>
+        /// <remarks>
+        /// Deleting a member will also delete all enrollments and study enrollments associated with said
+        /// member.
+        /// </remarks>
+        [HttpDelete("{id}")]
+        public async Task<IActionResult> DeleteMember(uint id, CancellationToken cancellationToken)
+        {
+            Member? member = await db.Members.FindAsync(id, cancellationToken);
+            if (member == null) return NotFound();
+
+            db.Members.Remove(member);
+            await db.SaveChangesAsync(cancellationToken);
+
+            return NoContent();
+        }
+
+        // PATCH: api/members/5/studentnumber
+        /// <summary>
+        /// Updates a member's student number.
+        /// </summary>
+        /// <param name="id">The id of the member to update.</param>
+        /// <param name="newStudentNumber">The new student number for the member.</param
+        /// <returns>No content.</returns>
+        [HttpPatch("{id}/studentnumber")]
+        public async Task<IActionResult> PatchMemberStudentNumber(uint id, uint newStudentNumber, CancellationToken cancellationToken)
+        {
+            Member? member = await db.Members.FindAsync(id, cancellationToken);
+            if (member == null) return NotFound();
+
+            member.StudentNumber = newStudentNumber;
+
+            await db.SaveChangesAsync(cancellationToken);
+
+            return NoContent();
+        }
+
+        // PATCH: api/members/5/firstName
+        /// <summary>
+        /// Updates a member's first name.
+        /// </summary>
+        /// <param name="id">The id of the member to update.</param>
+        /// <param name="newFirstName">The new first name for the member.</param>
+        /// <returns>No content.</returns>
+        [HttpPatch("{id}/firstName")]
+        public async Task<IActionResult> PatchMemberFirstName(uint id, string newFirstName, CancellationToken cancellationToken)
+        {
+            Member? member = await db.Members.FindAsync(id, cancellationToken);
+            if (member == null) return NotFound();
+
+            member.FirstName = newFirstName;
+
+            await db.SaveChangesAsync(cancellationToken);
+
+            return NoContent();
+        }
+
+        // PATCH: api/members/5/lastName
+        /// <summary>
+        /// Updates a member's last name.
+        /// </summary>
+        /// <param name="id">The id of the member to update.</param>
+        /// <param name="newLastName">The new last name for the member.</param
+        /// <returns>No content.</returns>
+        [HttpPatch("{id}/lastName")]
+        public async Task<IActionResult> PatchMemberLastName(uint id, string newLastName, CancellationToken cancellationToken)
+        {
+            Member? member = await db.Members.FindAsync(id, cancellationToken);
+            if (member == null) return NotFound();
+
+            member.LastName = newLastName;
```

Comment:
No String.EmptyOrWhiteSpace()? (See the other comment about creating an issue about form validation missing everywhere)